pallets/gear/src/manager/mod.rs

```diff
@@ -51,12 +51,15 @@ mod task;
 pub use journal::*;
```

Comment:
https://github.com/gear-tech/gear/blob/23aa9ef2c84062056589d0b54250dce92b6cef1b/pallets/gear/src/manager/task.rs#L145-L147
This seems tricky for me, I'd go with renaming into `remove_gas_reservation_impl` for manager's not-trait fn cause it's not intuitive atm and MAY depend on compiler (which fn to call, probability of inf recursion)